futurenhs.app/pages/groups/[groupId]/update/index.page.tsx

```diff
@@ -99,6 +99,25 @@ export const getServerSideProps: GetServerSideProps = withUser({
                                     themes[group.data.themeId]
                                         ? [group.data.themeId]
                                         : [defaultThemeId],
+                                GroupPrivacy: ['public']
+                            }
+                            /**
+                             * TODO: GroupPrivacy/groupIsPrivate to be determined by API response. 
+                             */
+                            const groupIsPrivate: boolean = false
+
+                            /**
+                             * Hides group is public checkbox if group is already private
+                             */
+                            if (groupIsPrivate) {
+
+                                const groupPrivacyField = form.steps[0]?.fields?.find((field) => field.name === 'GroupPrivacy')
```

Comment:
I think we need a generic form selector that will find and return a field by name